SmartDeviceLink/SDLStreamingVideoLifecycleManager.m

```diff
@@ -122,6 +123,7 @@ - (instancetype)initWithConnectionManager:(id<SDLConnectionManagerType>)connecti
     _useDisplayLink = configuration.streamingMediaConfig.enableForcedFramerateSync;
     _screenSize = SDLDefaultScreenSize;
     _backgroundingPixelBuffer = NULL;
+    _showVideoBackgroundDisplay = configuration.showVideoBackgroundDisplay;
```

Comment:
```suggestion
    _showVideoBackgroundDisplay = configuration.streamingMediaConfig.showVideoBackgroundDisplay;
```

Reply:
Thanks. Fixed.

Reply:
add test code in `SDLStreamingMediaConfigurationSpec`.